spectec/test-prose/doc/valid/instructions-in.rst

```diff
@@ -65,10 +65,7 @@ $${rule-prose: valid/cvtop}
 
 \
 
-$${rule+: 
-  Instr_ok/reinterpret
-  Instr_ok/convert-*
-}
+$${rule+: Instr_ok/cvtop-*}
```

Comment:
This test was supposed to test splices with multiple rules listed, so preferably change it to
```suggestion
$${rule+:
  Instr_ok/cvtop-reinterpret
  Instr_ok/cvtop-convert-*
}
```